### ISSUE TITLE
AVT 3 - React MultiSelect multiple navigation issues while using VO

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4197]
## Environment
macOS 10.14.6
Chrome Version 77.0.3865.90
VoiceOver

## Description
This issue applies to both the default MultiSelect and the MultiSelect w/initial selected items:
1. With VO active tab to the MultiSelect
2. VO does not announce the Multiselect label
**Expected Result:** The label should be announced
3. Use the enter key to open the menu, but the menu fails to open
**Expected Result:** The enter key should open the menu
4. Use the space key to open the menu, works as expected.
5. Arrow to a selection and press Space bar to check the box, VO announces the option text but does allow the user to make a selection. 
**Expected Result:** User should be able to check 1 or more checkboxes.

**Comment by carmacleod** [comment]
These checkboxes can currently be checked with the enter key... however, the [expected behavior is for them to be  checked with the space key](http://w3c.github.io/aria-practices/#keyboard-interaction-5).

Note that I don't think you want to keep allowing the Enter key to check/uncheck the checkboxes, because what if the control is in a dialog that has a default button that is activated on Enter?

So I think the check/uncheck behavior has to be on space key only, which is how the regular Carbon checkboxes operate already (i.e. Space only, not Enter). Thanks!

Storybook link for multiselect: http://react.carbondesignsystem.com/?path=/story/multiselect--default
Storybook link for regular checkboxes: http://react.carbondesignsystem.com/?path=/story/checkbox--checked

**Comment by dakahn** [comment]
closed by https://github.com/carbon-design-system/carbon/pull/5373
